src/main/java/com/synopsys/integration/detect/configuration/DetectConfigurationFactory.java

```diff
@@ -473,6 +473,16 @@ private Path getPathOrNull(NullablePathProperty property) {
         return detectConfiguration.getValue(property).map(path -> path.resolvePath(pathResolver)).orElse(null);
     }
 
+    private Path getPathOrDefault(DetectProperty<NullablePathProperty> property, Path defaultPath) {
+        Path path = getPathOrNull(property.getProperty());
+        return path != null ? path : defaultPath;
+    }
+
+    private Path getPathOrDefault(NullablePathProperty property, Path defaultPath) {
```

Comment:
If there exists a default value, then the property shouldn't be a `NullablePathProperty`.

Reply:
I checked out the branch to see what used this and it's unused. Probably best to remove these two methods.

Reply:
Yeah just forgot to get rid of these.